en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -16,7 +18,7 @@ In this section, let's develop the application that retrieves COVID-19-related s
 
 1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev).
 
-2. Expand the drop-down menu for projects and click **+ Create New**.
+2. In the **Projects** field, select **+ Create New** from the list.
```

Comment:
Based on the style guide we would have to go with the following I suppose:
In the **Projects** list, select **+ Create New**.

Reply:
Changed as suggested.